dali/kernels/normalize/normalize_cpu.h

```diff
@@ -35,36 +35,38 @@ namespace kernels {
 namespace normalize_impl {
 
 template <typename Out, typename In, typename Param>
-void normalize(Out *out, const In *in, int64_t count, const Param *mean, const Param *inv_stddev) {
+void normalize(Out *out, const In *in, int64_t count,
+               const Param *mean, const Param *scale, float shift) {
```

Comment:
```suggestion
               const Param *mean, const Param *scale, Param shift) {
```
?

Reply:
Nice cactch. I don't know if I need this `Param` at all, but if it's there, then yes.